src/Native/Unix/configure.cmake

```diff
@@ -174,6 +174,20 @@ check_cxx_source_compiles(
 
 check_cxx_source_compiles(
     "
+    #include <dirent.h>
+    int main(void)
+    {
+        DIR* dir;
+        struct dirent* entry;
+        struct dirent* result;
+        readdir_r(dir, entry, &result);
```

Comment:
This compiles with the current settings. We need to put this into the configuration file if we want this to work:

```
set(CMAKE_REQUIRED_FLAGS -Werror)
```